src/Microsoft.Azure.WebJobs.Host/Bindings/BindingFactory.cs

```diff
@@ -47,6 +47,19 @@ public IConverterManager ConverterManager
         }
 
         /// <summary>
+        /// Creating a type filter predicate around another rule. Filter doubles as a validator if it throws.  
+        /// </summary>
+        /// <param name="predicate">type predicate. Only apply inner rule if this predicate as applied to the user parameter type is true. </param>
+        /// <param name="innerRule">Inner rule. </param>
+        /// <returns></returns>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1822:MarkMembersAsStatic")]
```

Comment:
nit: can you add exclusions to the global file rather than inline in source? That's the pattern we generally practice across these repos.